polaris-react/src/components/Columns/Columns.tsx

```diff
@@ -15,6 +15,7 @@ type ColumnsAlias = 'oneThird' | 'oneHalf' | 'twoThirds';
 type ColumnsType = number | string | ColumnsAlias[];
 type Columns = ResponsiveProp<ColumnsType>;
 type Gap = ResponsiveProp<SpacingSpaceScale>;
+type ColumnsAlignItems = 'start' | 'end' | 'center';
```

Comment:
Should this be responsive?

Reply:
Good question. I'm inclined to leave it as is for now and add in responsive if the use case comes up.

Reply:
➕ to keeping it simple and adding responsive support if needed.